region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -287,6 +287,67 @@
       "flashSuitChecked": true,
       "note": "Assumes stopping two tiles before the gap, to have enough space run and gain enough speed to jump over the gap without needing to turn around first."
     },
+    {
+      "link": [1, 1],
+      "name": "Leave Spinning",
+      "requires": [],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 20,
+            "openEnd": 1
+          }
+        }
+      }
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave With Mockball",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 20,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 5,
+            "openEnd": 1
+          }
+        }
+      },
+      "collectsItems": [5],
+      "devNote": "The item could be avoided by using a shorter landing runway and enough speed"
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave Space Jumping",
+      "requires": [
+        "canPreciseSpaceJump"
```

Comment:
Yeah, added "collectsItems" in a bunch more places now, to be safe. I know there would still be more ways to avoid the item, but I'm not sure it's worth the hassle at this point to fully model it, given the amount of duplication of the strats that this would need.